app/helpers/payments_helper.rb

```diff
@@ -1,2 +1,32 @@
 module PaymentsHelper
+  def generate_uuid
+     SecureRandom.uuid
+  end
+def eligible_for_email_users
+    group_ids = Setting.plugin_redmine_leaves['eligible_for_email_notification']
+    if group_ids.blank?
```

Comment:
if no group is selected, then nil should be returned and no email notification sent